RecoMTD/DetLayers/src/ETLDetLayerGeometryBuilder.cc

```diff
@@ -176,15 +178,17 @@ MTDSectorForwardDoubleLayer* ETLDetLayerGeometryBuilder::buildLayerNew(int endca
     }
 
     if (!backGeomDets.empty()) {
-      std::sort(backGeomDets.begin(), backGeomDets.end(), orderGeomDets);
+      //std::sort(backGeomDets.begin(), backGeomDets.end(), orderGeomDets);
```

Comment:
Why do you want to keep this line as a comment?

Reply:
this is just a minor leftover from the development, to be removed